test/manual-test-examples/webgl/geometryQuads/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title></title>
+  <link rel="stylesheet" href="../../styles.css">
+  <script language="javascript" type="text/javascript" src="../../../../lib/p5.js"></script>
+  <script language="javascript" type="text/javascript" src="sketch.js"></script>
+  <script src="../stats.js"></script>
```

Comment:
It appears that the loader for stats.js [included in the manual tests](https://github.com/processing/p5.js/blob/main/test/manual-test-examples/webgl/stats.js) is no longer functioning. This should be fixed but also the library can be omitted from your test since it uses `noLoop()`.

Reply:
Looks like the issue is just that the find-and-replace from `master` to `main` accidentally affected the path to the stats script, which is included from the mrdoob stats.js library. I've replaced `main` with the current latest commit hash on that repo, so if they ever do any restructuring over there on the master branch, it won't accidentally break our tests.